raster/r.grow.distance/main.c

```diff
@@ -190,7 +203,41 @@ int main(int argc, char **argv)
     dist_name = opt.dist->answer;
     val_name = opt.val->answer;
 
-    if ((invert = flag.n->answer)) {
+    dist_row_max = val_row_max = NULL;
+    mindist = -1;
+    if (opt.min->answer) {
+	if (sscanf(opt.min->answer, "%lf", &mindist) != 1) {
+	    G_warning(_("Invalid %s value '%s', ignoring."),
+		      opt.min->key, opt.min->answer);
+
+	    mindist = -1;
+	}
+    }
+
+    maxdist = -1;
+    if (opt.max->answer) {
+	if (sscanf(opt.max->answer, "%lf", &maxdist) != 1) {
+	    G_warning(_("Invalid %s value '%s', ignoring."),
+		      opt.max->key, opt.max->answer);
+
+	    maxdist = -1;
+	}
+    }
+
+    if (mindist > 0 && maxdist > 0 && mindist >= maxdist) {
+	G_fatal_error(_("%s must be smaller than %s."),
```

Comment:
Add a translator comment before this message.

Reply:
I don't know what you mean with "translator comment", please explain.

Reply:
I suppose what’s described in https://trac.osgeo.org/grass/wiki/Submitting/C#Messages.

Reply:
Tried to add a translator comment, ok @marisn ?

Reply:
Seems good. Sometimes simply it is hard to understand how to translate a phrase when it is unknown what %s stands for in the final message.